Sources/macro.js

```diff
@@ -31,7 +31,7 @@ export function getStateArrayMapFunc(item) {
 
 export function obj(publicAPI = {}, model = {}) {
   const callbacks = [];
-  model.mtime = model.mtime || ++globalMTime;
+  model.mtime = (model.hasOwnProperty('mtime') ? model.mtime : ++globalMTime);
```

Comment:
I would do it like that instead

model.mtime = Number.isInteger(model.mtime) ? model.mtime : ++globalMTime;